cogs/tibia.py

```diff
@@ -1932,6 +1932,32 @@ def __unload(self):
         print("cogs.tibia: Cancelling pending tasks...")
         self.news_announcements_task.cancel()
 
+    @checks.is_mod()
+    @commands.command(name="respawnarea", aliases=['resparea'])
+    async def respawn_area(self, ctx: NabCtx):
+        """Announces the respawn area if there is a channel set. Area is chosen by mods."""
+        guild = ctx.guild
+        channel_id = await get_server_property(ctx.pool, guild.id, "channel_respawn_area")
+        if channel_id == 0:
```

Comment:
This won't be the default value for the property, the default value is `None`. If you want to have a default value for when the value doesn't exist, use the `default` parameter for the function `get_server_property`

Reply:
- [ ] Fix default return of `get_server_property` in `/resparea`

Reply:
Comparing with default `None` instead of `0` now.